packages/fields/src/fields/featured-image/featured-image-view.tsx

```diff
@@ -22,14 +23,37 @@ export const FeaturedImageView = ( {
 		},
 		[ mediaId ]
 	);
-	const url = media?.source_url;
+
+	// Thumbnail will exist almost for sure, but better have a fallback anyway.
+	const thumbnailURL =
+		media?.media_details?.sizes?.thumbnail?.source_url || media?.source_url;
+	const url = view ? media?.source_url : thumbnailURL;
+
+	// @ts-ignore layout exists on ViewGrid; not sure what the problem is here.
```

Comment:
It looks like it's because only [ViewGrid](https://github.com/WordPress/gutenberg/blob/try/dataviews-responsive-image-grid/packages/dataviews/src/types.ts#L377) has the `previewSize` prop.

I guess something like this would work:

```ts
	const columnNumberSetting =
		'grid' === view.type ? view.layout?.previewSize || 0 : 0;
```